app/demoservice/templates/base.html

```diff
@@ -27,13 +28,21 @@
     </nav>
   </div>
 </header>
-{% if messages %}
-  <ul class="messages">
+  {% if messages %}
     {% for message in messages %}
-      <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message }}</li>
+    <div class="p-strip">
```

Comment:
We probably don't want a new strip per notification. Can you move the strip and row out side of the loop with an if statement please.

Reply:
But I don't want an empty strip element unless there is a message as it'll push all content dow the page. AFAIK, there'll only ever be one notification at any one time which is why I wrapped it all in the conditional.

Reply:
Currently there is only one notification but it could be any number.
I see the strip is in the `if`. I think the `for` loop can be moved inside the `col-12`.